android/src/org/ppsspp/ppsspp/NativeActivity.java

```diff
@@ -788,6 +792,14 @@ public void onConfigurationChanged(Configuration newConfig) {
 		}
 	}
 
+	@Override
+	public void onMultiWindowModeChanged(boolean isInMultiWindowMode, Configuration newConfig) {
+		// onConfigurationChanged not called on multi-window change
+		Log.i(TAG, "onMultiWindowModeChanged: isInMultiWindowMode = " + isInMultiWindowMode);
+		super.onMultiWindowModeChanged(isInMultiWindowMode, newConfig);
+		updateDisplayMeasurements();
```

Comment:
Hm, in theory we want the `densityDpi` from newConfig, though we don't seem to call `NativeApp.setDisplayParameters` in `onConfigurationChanged` so it's probably already wrong in cases.

-[Unknown]